skaha/src/main/java/org/opencadc/skaha/SkahaAction.java

```diff
@@ -315,7 +315,6 @@ protected String callHarbor(String idToken, String harborHost, String project, S
         } else {
             harborURL = new URL("https://" + harborHost + "/api/v2.0/projects/" + project + "/repositories/"
                 + repo + "/artifacts?detail=true&with_label=true&page_size=-1");
-            message = "artifacts";
```

Comment:
Why was this removed?

Reply:
Good eye. It was accidentally removed. Added back.